backend/Squirrel.Core/Squirrel.Core.Common/DTO/Auth/AccessToken.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Squirrel.Core.Common.DTO.Auth
+{
```

Comment:
it's better to use file-scoped namespaces

Reply:
yes, updated